nominal/core/_clientsbunch.py

```diff
@@ -25,52 +28,40 @@
 @dataclass(frozen=True)
 class ClientsBunch:
     auth_header: str
-    run: scout.RunService
-    upload: upload_api.UploadService
+
+    attachment: attachments_api.AttachmentService
     authentication: authentication_api.AuthenticationServiceV2
-    ingest: ingest_api.IngestService
     catalog: scout_catalog.CatalogService
-    attachment: attachments_api.AttachmentService
-    compute_representation: scout_compute_representation_api.ComputeRepresentationService
     checklist: scout_checks_api.ChecklistService
-    video: scout_video.VideoService
-    logset: datasource_logset.LogSetService
-    units: scout_units_api.UnitsService
+    compute_representation: scout_compute_representation_api.ComputeRepresentationService
+    dataexport: scout_dataexport_api.DataExportService
```

Comment:
dataexport was added in this PR, the other changes are due to alphabetizing this long list